OEL-1326: Cleaning patterns & paragraphs from OEL-755

escuriola opened the pull request:
Jira issue(s):
https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-1326

escuriola committed:
OEL-1326: Remove from testing date and other non-useful data. Remove also some asserts with estetical classes.

escuriola committed:
OEL-1326: Not using a variable for code used only once.

donquixote commented on modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/ListingParagraphsTest.php:
```diff
@@ -174,7 +168,6 @@ protected function createListItems(File $image_file, Node $node): array {
           'alt' => 'Alt for image ' . $i,
           'target_id' => $image_file->id(),
         ],
-        'field_oe_date' => '2011-11-13',
```
From the other PR [oe_bootstrap_theme:191](https://github.com/openeuropa/oe_bootstrap_theme/pull/191):

> you don't need all this, you'r not asserting the paragraph content in this test, and that is ok, you don't need date, you dont need image, you dont need most, add just minimal and use a random function instead where needed

donquixote replied:
Not sure if anything else should be removed here. But I am going to assume it is ok.

brummbar committed:
OEL-1326: Temporary ticket patch to allow merging.

Multiple tickets were merged in oe_bootstrap_theme but not the corresponding
in whitelabel.

brummbar committed:
OEL-1326: Use correct class for card links.
